src/analysis/normed_space/bounded_linear_maps.lean

```diff
@@ -44,6 +45,9 @@ lemma is_linear_map.with_bound
 
 namespace is_bounded_linear_map
 
+def to_linear_map {f : E → F} (h : is_bounded_linear_map f) : E →ₗ[k] F :=
```

Comment:
`f` should be explicit here, because `h` gets omitted in term printouts, so it's really confusing looking at `to_linear_map _`